postinstall.sh

```diff
@@ -0,0 +1,5 @@
+git clone https://github.com/mapbox/node-srs ../node-srs
+pushd $(pwd)/../node-srs > /dev/null
+npm install --build-from-source
+popd > /dev/null
+ln -s $(pwd)/../node-srs node_modules/node-srs
```

Comment:
This is a change from the original instructions — it's a quick fix if you want this to be a `cp` instead of `ln -s`.